go.mod

```diff
@@ -5,7 +5,7 @@ go 1.22
 require (
 	github.com/gobuffalo/flect v1.0.2
 	github.com/onsi/ginkgo/v2 v2.19.1
-	github.com/onsi/gomega v1.34.0
+	github.com/onsi/gomega v1.34.1
```

Comment:
It appears that there is an external Go module update included in this PR. Should we clear this update within the PR?